content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md

```diff
@@ -15,97 +15,143 @@ tags:
 {{< include file="/_includes/CommunityAppsLegacy.md" >}}
 {{< include file="/_includes/CommunityAppsContribute.md" >}}
 
+The TrueNAS SCALE Netdata app provides an easy way to install and access the Netdata infrastructure monitoring solution. 
+SCALE deploys the Netdata app in a Kubernetes container using the Helm package manager. 
+After successfully deploying the app, you can access the Netdata web portal from SCALE. 
+The Netdata web portal opens on the local dashboard, and where you can create new dashboards, add plugins, metric databases, physical and virtual systems, containers, and other cloud deployments you want to monitor. 
+The portal also provides access to the Netdata Cloud sign-in screen.
+
 ## Before You Begin
+The SCALE Netdata app does not require advance preparation.
 
-Before using SCALE to install the Netdata application you need to configure TrueNAS SCALE storage for the Netdata application to use.
+You can allow SCALE to automatically create storage volumes for the Netdata app or you can create specific datasets to use for configuration, cache, and library storage and extra storage volumes in the container pod.
+If using specific datasets, create these before beginning the app installation.
 
-Verify the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account has sudo permissions enabled.
+The administrator account must have sudo permissions enabled. 
+To verify, go to **Credentials > Local User**. 
+Click on the administrator user (e.g., admin), then click **Edit**. Scroll down to the sudo permissions.
```

Comment:
What should they check for or enable here? **Allow all sudo commands**, **Allow all sudo commands with no password**, both?